.github/workflows/release.yaml

```
@@ -13,6 +13,17 @@ jobs:
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           fetch-depth: 0
+          
+      - name: Set tag output
+        id: tag
+        run: echo "tag_name=${GITHUB_REF#refs/*/}" >> "$GITHUB_OUTPUT"
+
+      - name: Install tejolote
+        uses: kubernetes-sigs/release-actions/setup-tejolote@dd08496c83441d6477114cc0555b96d404dacff7 # v0.1.2
+
+      - run: |
+          tejolote attest --artifacts github://kubernetes-sigs/karpenter/${{ steps.tag.outputs.tag_name }} github://kubernetes-sigs/karpenter/"${GITHUB_RUN_ID}" --output karpenter.intoto.json --sign
```

```suggestion
          tejolote attest --artifacts github://kubernetes-sigs/karpenter/${{ github.ref_name }} github://kubernetes-sigs/karpenter/"${{ github.run_id }}" --output karpenter.intoto.json --sign
```

You can get the tag directly using the `${{ github.ref_name }}`. Updated the other reference for consistency